src/module/vue/components/character-sheet-tabs/ironsworn-main.vue

```diff
@@ -63,7 +63,7 @@ import Bonds from '../bonds.vue'
 import OrderButtons from '../order-buttons.vue'
 import Asset from '../asset/asset.vue'
 import BtnCompendium from '../buttons/btn-compendium.vue'
-import ProgressBox from '../progress/progress-box.vue'
+import ProgressListItem from '../progress/progress-list-item.vue'
```

Comment:
Doesn't look like this is used here.

```suggestion
```

Reply:
hm. i renamed `ProgressBox` to `ProgressListItem` to disambiguate it (since a progress box can also refer to a box of progress on a track). is `ProgressBox` unused here on `main`?